packages/babel-preset-react-app/dependencies.js

```diff
@@ -87,8 +87,7 @@ module.exports = function (api, opts) {
           // Allow importing core-js in entrypoint and use browserlist to select polyfills
           useBuiltIns: 'entry',
           // Set the corejs version we are using to avoid warnings in console
-          // This will need to change once we upgrade to corejs@3
-          corejs: 3,
+          corejs: 3.5,
```

Comment:
According to https://github.com/babel/babel/blob/90a9103e5560a49f98be88d531ae3992fdcafe0b/packages/babel-preset-env/src/types.js#L12 this should be a string, e.g. `'3.5'`, not a number

```suggestion
          corejs: '3.5',
```